src/components/TradingTerminal/TradingView/TradingViewEdit.js

```diff
@@ -319,6 +342,17 @@ const TradingViewEdit = (props) => {
               />
             )}
           </Box>
+          {storeUserData && storeUserData.isAdmin && positionId && (
+            <Box marginTop={2} maxWidth="300px" minWidth="300px">
```

Comment:
Do we really need to specify the width there?

Reply:
we dont but I wanted to do it here instead of in css.